pandas/core/reshape/merge.py

```diff
@@ -591,6 +591,8 @@ def __init__(
     ):
         _left = _validate_operand(left)
         _right = _validate_operand(right)
+        if how == "cross":
+            _left, _right, how, on = self._create_cross_configuration(_left, _right, on)
```

Comment:
return the new column here (in addition to the other values)

Reply:
Done